scripts/upload-resources-to-github

```diff
@@ -3,6 +3,11 @@ set -euo pipefail
 
 SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
 VERSION=$(make -s -f $SCRIPTPATH/../Makefile version)
+BUILD_DIR=$SCRIPTPATH/../build/k8s-resources/$VERSION
+INDV_RESOURCES_DIR=$BUILD_DIR/indv-resources
+TAR_RESOURCES_FILE=$BUILD_DIR/aws-node-termination-handler-k8s-resources.tar
+AGG_RESOURCES_YAML=$BUILD_DIR/aws-node-termination-handler.yaml
```

Comment:
I still think it would be a little more cogent to name the tarball `individual-resources.tar` and the aggregated YAML file `all.yaml` or `all-resources.yaml`. Thoughts?

Reply:
ah yes, I see what you mean. I overlooked the additional stutter in your first comment. I'll update